<?xml version="1.0"?>
<launch>
    
  <rosparam file="$(find robot_labinm_navigation)/config/controllers.yaml" command="load"/>
  <rosparam file="$(find robot_labinm_navigation)/config/joint_limits.yaml" command="load"/>
     
  <node name="robot_hardware_interface" pkg="robot_labinm_navigation" type="mobile_robot_hardware_interface" output="screen">
    <remap from="/mobile_robot/mobile_base_controller/cmd_vel" to="/cmd_vel"/>
  </node>
  
  <node name="controller_spawner" pkg="controller_manager" type="spawner" respawn="false" output="screen"
        args="
            /mobile_robot/joints_update
            /mobile_robot/mobile_base_controller" >
  </node>
  
  <node name="joy_node" pkg="joy" type="joy_node" />
  
  <node name="robot_labinm_teleop" pkg="robot_labinm_navigation" type="robot_labinm_teleop.py" output="screen" />
  
</launch>
